src/main/resources/init.sh

```diff
@@ -93,11 +93,14 @@ stop)
         exit 0
     fi
 ;;
+console)
+    service/bin/$SERVICE $ARGS
```

Comment:
I think you probably ought to check to see if the service is running and failing if it's running already. I also suspect you want to write a pid file and attach to the process rather than just execute the command, in particular because that allows start/stop to fail if the process is being run from console.